lib/ronin/support/text/text_score/generate_glyphs.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+require 'awesome_print'
+require 'erb'
+
+# Computes n-glyphs from a corpus of text
+class NGlyphFrequencyAnalyzer
```

Comment:
I would strongly discourage putting internal utility code into `lib/` as any other gem can require that code. Instead, you should define this as a `rake` task to generate the files, ideally using Rake's `file` task.

Also, it's not a good idea to define classes at the top-level within the `lib/` directory.